hack/dev/userdata/.master-1.tpl.yaml

```diff
@@ -2,14 +2,19 @@ services:
   init:
     cni: flannel
   kubeadm:
+    certificateKey: verysimplesecretyoucanguess.....
     configuration: |
       apiVersion: kubeadm.k8s.io/v1beta1
       kind: InitConfiguration
       bootstrapTokens:
       - token: 1qbsj9.3oz5hsk6grdfp98b
         ttl: 0s
       nodeRegistration:
+        name: master-1
```

Comment:
without it, node hostname gets injected into config so all the containers get the same node name

node name doesn't resolve inside container (separate issue to be fixed), but this brings us one step closer